library/data/tags/build.gradle

```diff
@@ -17,4 +20,21 @@ qslModule {
 			testmodOnly("command")
 		}
 	}
+	entrypoints {
+		events {
+			values = ["org.quiltmc.qsl.tag.impl.TagRegistryImpl"]
+		}
+		client_init {
+			values = ["org.quiltmc.qsl.tag.impl.client.ClientQuiltTagsMod"]
+		}
+	}
+	injectedInterface("net/minecraft/class_3494") {
+		values = ["org/quiltmc/qsl/tag/api/QuiltTag"]
+	}
+	injectedInterface("net/minecraft/class_3494\$class_3495") {
+		values = ["org/quiltmc/qsl/tag/api/QuiltTagBuilder"]
+	}
+	injectedInterface("net/minecraft/class_6862") {
+		values = ["org/quiltmc/qsl/tag/api/QuiltTagKey"]
+	}
```

Comment:
idea:
```suggestion
	injectedInterfaces {
		"net/minecraft/class_3494" = ["org/quiltmc/qsl/tag/api/QuiltTag"]
		"net/minecraft/class_3494\$class_3495" = ["org/quiltmc/qsl/tag/api/QuiltTagBuilder"]
		"net/minecraft/class_6862" = ["org/quiltmc/qsl/tag/api/QuiltTagKey"]
	}
```

Reply:
There is probably a way to abuse Closures and the map construction syntax to accomplish that, but
1) that's internal API once gradle 8 releases
2) I don't want to put in the work to make this look pretty, when I'm hoping to replace it with an annotation processor anyway in the future.